CHANGELOG_PENDING.md

```diff
@@ -3,3 +3,7 @@
 ### SDK Enhancements
 
 ### SDK Bugs
+* `rest`: Remove unnecessary path normalization behavior.
+    * This behavior would incorrectly mutate request paths with URI-encoded characters, potentially resulting in misrouted requests.
```

Comment:
Two spaces here to indent instead of 4. IIRC the code that handles this is incredibly picky, best to be safe.